### DISCUSSION TITLE
EP and Windows 11 update KB5023778 problem

### DISCUSSION THREAD

**Discussion by Muyfa666** [valinet/ExplorerPatcher#1287] [Q&A]
Whenever I install Windows 11 update KB5023778, Windows get stuck at the desktop, flashing indefinetly, without any menus etc working. If I remove EP or the update, everything works fine.

Any workaround or fix for this?

**Comment by topia** [comment]
There is no workaround or fix attempt until Microsft releases fully (as Patch Tuesday), at least. Moreover, devs may require some time to support them.
https://github.com/valinet/ExplorerPatcher/wiki/Symbols#how-does-explorerpatcher-obtain-the-necessary-symbol-files

> Symbols are usually NOT made available for the following types of builds:
>
> - builds shipped via the Beta and Release Preview channel of the Windows Insider Program
> - "C" and "D" stable OS builds - these are builds labeled as "Preview" and offered via Windows Update in the second part of a month, representative of what will generally arrive on the next Patch Tuesday; these are optional, the user has to explicitly click "Download and install" to have them applied on their computer, they are never automatically installed by Windows Update

**Comment by Muyfa666** [comment]
This got way to too technical way too fast... ;-)

At least it's a known issue that may actually get fixed?
